pyqt-apps/siriushla/as_rf_control/control.py

```diff
@@ -213,6 +213,15 @@ def _statusLayout(self):
         self.pb_llrfreset.setStyleSheet(
             '#pb_llrfreset{min-width:25px; max-width:25px; icon-size:20px;}')
 
+        # FDL
+        self._ld_fdl = QLabel('<h4>FDL</h4>', self, alignment=Qt.AlignLeft)
```

Comment:
Maybe the group can be "Auxiliary windows", to be possible to group any other GUI in the future